optuna/storages/_journal/file.py

```diff
@@ -189,7 +189,7 @@ def read_logs(self, log_number_from: int) -> List[Dict[str, Any]]:
 
     def append_logs(self, logs: List[Dict[str, Any]]) -> None:
         with get_lock_file(self._lock):
-            what_to_write = "\n".join([json.dumps(log) for log in logs]) + "\n"
+            what_to_write = os.linesep.join([json.dumps(log) for log in logs]) + os.linesep
```

Comment:
The Python reference does not recommend `os.linesep` when we writing text files.
> Do not use os.linesep as a line terminator when writing files opened in text mode (the default); use a single '\n' instead, on all platforms.

We may stick to `\n` or open the file as binary.

https://docs.python.org/3/library/os.html#os.linesep

Reply:
Oh, thanks. Good catch 👍  I will revert this line.